RE-BASELINE: REGRESSION(267516@main): [ macOS wk2 ] editing/input/caret-at-the-edge-of-contenteditable.html is constantly failing.

bls1999 opened the pull request:
#### 292595cd441a4974764cd306ac2433cf66071b41
<pre>
RE-BASELINE: REGRESSION(267516@main): [ macOS wk2 ] editing/input/caret-at-the-edge-of-contenteditable.html is constantly failing.
<a href="https://bugs.webkit.org/show_bug.cgi?id=267461">https://bugs.webkit.org/show_bug.cgi?id=267461</a>
<a href="https://rdar.apple.com/120913774">rdar://120913774</a>

Unreviewed test re-baseline.

Correcting errant baseline migration from OpenSource expectations migration at 267516@main.

* LayoutTests/platform/mac-ventura-wk1/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Renamed from LayoutTests/platform/mac-monterey/editing/input/caret-at-the-edge-of-contenteditable-expected.txt.
* LayoutTests/platform/mac-ventura-wk2/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Renamed from LayoutTests/platform/mac-ventura-wk2/editing/caret/caret-at-the-edge-of-contenteditable-expected.txt.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac-wk2/editing/input/caret-at-the-edge-of-contenteditable-expected.txt:
* LayoutTests/platform/mac/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/273087@main">https://commits.webkit.org/273087@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/69edf056105fa20dafa097a2ccf5ecbca07c0d4b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/34234 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/13043 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/36225 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/36923 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/31016 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/15430 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/10173 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/30042 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/34746 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/11044 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/30546 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/9641 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/9751 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/30584 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/38213 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/31108 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/30901 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/35828 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/9872 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/7737 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/33744 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/11659 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/10431 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/4394 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/10692 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

robert-jenner reviewed (changes requested):
I checked out your PR on a machine running Ventura and ran it. The result I got was that it passes on Ventura WK1, but it still fails on Ventura WK2. So it looks like you will need to set a specific rebaseline for Ventura WK2 in order to fix it there. But so far this change did fix it for Ventura WK1.

bls1999 committed:
RE-BASELINE: REGRESSION(267516@main): [ macOS wk2 ] editing/input/caret-at-the-edge-of-contenteditable.html is constantly failing.
https://bugs.webkit.org/show_bug.cgi?id=267461
rdar://120913774

Unreviewed test re-baseline.

Correcting errant baseline migration from OpenSource expectations migration at 267516@main.

* LayoutTests/platform/mac-ventura-wk1/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Renamed from LayoutTests/platform/mac-monterey/editing/input/caret-at-the-edge-of-contenteditable-expected.txt.
* LayoutTests/platform/mac-ventura-wk2/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Renamed from LayoutTests/platform/mac-ventura-wk2/editing/caret/caret-at-the-edge-of-contenteditable-expected.txt.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac-wk2/editing/input/caret-at-the-edge-of-contenteditable-expected.txt:
* LayoutTests/platform/mac/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.

Canonical link: https://commits.webkit.org/273087@main
